utils/telemetry/events.go

```diff
@@ -0,0 +1,37 @@
+package telemetry
+
+import (
+	"time"
+)
+
+// REALMC-7243 same as the new cli Segment tracking
```

Comment:
we can just remove these comments (sometimes its nice to mention when a file was copy/pasted like this), but it's fine here without that